src/features/likes/likes-slice.jsx

```diff
@@ -21,8 +22,8 @@ export const addToLikes = createAsyncThunk(
           },
         }
       );
-      console.log("25:response", res.data.likes);
-      return res.data.likes;
+
+      return response.data.likes;
     } catch (error) {
       console.log(error);
```

Comment:
You can return `error.response` that will give you the error message. And later in cases you can use toast to display that message

Reply:
okay will implement later